### PR TITLE
Add Trino tenth birthday post

### PR DESCRIPTION
**Pull request by bitsondatadev**
The tenth birthday blog is ready to be reviewed.

### CODE REVIEW

File: _posts/2022-08-08-trino-tenth-birthday.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+one location using ANSI SQL! Since the projects split, Trino has skyrocketed in
+development from the original project and has added an array of features that
+we’ve listed out in the evolution of the Trino architecture blog post.
+
+![](/assets/blog/trino-tenth-birthday/trajectory.png)
+
+To really celebrate this milestone, we wanted to offer some exciting ways for
+you to learn more about Trino, and spin up Trino on your own system to play
+around with it. We have a list of blogs, project stats, and ways to get involved
+below. Starburst is also celebrating by offering free Trino birthday t-shirts
```
Comment (bitsondatadev) [inline _posts/2022-08-08-trino-tenth-birthday.md:43]: Add link from @monimiller when ready

File: _posts/2022-08-08-trino-tenth-birthday.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: "Happy 10th Birthday Trino!"
```
Comment (mosabua) [inline _posts/2022-08-08-trino-tenth-birthday.md:3]:
```suggestion
title: "Happy tenth birthday Trino!"
```

File: _posts/2022-08-08-trino-tenth-birthday.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+author: "Brian Olsen, Martin Traverso, Dain Sundstrom, David Phillips, Eric Hwang"
+excerpt_separator: <!--more-->
+---
+
+![](/assets/blog/trino-tenth-birthday/how-it-started-going.png)
+
+It’s inspiring and mindblowing to reflect on the ten year journey that has
+produced the community around Trino. Trino is the community-driven fork from
+Presto, the distributed big data query engine created at Facebook in 2012. We
```
Comment (mosabua) [inline _posts/2022-08-08-trino-tenth-birthday.md:12]:
```suggestion
Presto, the distributed big data SQL query engine created at Facebook in 2012. We
```

File: _posts/2022-08-08-trino-tenth-birthday.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+<iframe src="https://youtu.be/hPD95_-bZZw" width="800" height="500" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; 
+margin-bottom:5px; max-width: 100%;" allowfullscreen=""> 
+</iframe>
+
+As we watch the video and think back to the five years Presto and Trino shared,
+you begin to appreciate the organic development of the community, and the
+excitement around the solution space that the project brought to big data. As a
+baseline, Trino offers a faster and more interactive alternative to accessing
+data stored in HDFS via Hive. But the project didn’t stop there. Development of
+the SPI abstracted metadata and storage, among other interfaces to different
```
Comment (mosabua) [inline _posts/2022-08-08-trino-tenth-birthday.md:32]:
```suggestion
the SPI abstracted metadata and storage access to different
```

File: _posts/2022-08-08-trino-tenth-birthday.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+systems, making Trino a suitable system to query an entire data ecosystem from
```
Comment (mosabua) [inline _posts/2022-08-08-trino-tenth-birthday.md:33]:
```suggestion
systems, making Trino a suitable engine to query an entire data ecosystem from
```

File: _posts/2022-08-08-trino-tenth-birthday.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+one location using ANSI SQL! Since the projects split, Trino has skyrocketed in
+development from the original project and has added an array of features that
```
Comment (mosabua) [inline _posts/2022-08-08-trino-tenth-birthday.md:35]:
```suggestion
development from the original project and added an array of features that
```

File: _posts/2022-08-08-trino-tenth-birthday.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+we’ve listed out in the evolution of the Trino architecture blog post.
```
Comment (mosabua) [inline _posts/2022-08-08-trino-tenth-birthday.md:36]: link

File: _posts/2022-08-08-trino-tenth-birthday.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+
+![](/assets/blog/trino-tenth-birthday/trajectory.png)
+
+To really celebrate this milestone, we wanted to offer some exciting ways for
+you to learn more about Trino, and spin up Trino on your own system to play
+around with it. We have a list of blogs, project stats, and ways to get involved
+below. Starburst is also celebrating by offering free Trino birthday t-shirts
+when you complete their Space Quest League mission. Also don’t forget to attend 
+our annual Trino Summit in November!
```
Comment (mosabua) [inline _posts/2022-08-08-trino-tenth-birthday.md:45]: link to blog post on trino.io

File: _posts/2022-08-08-trino-tenth-birthday.md [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+* Subscribe to the [Trino YouTube channel](https://www.youtube.com/c/trinodb)
+* Follow us on the [trinodb Twitter account](https://twitter.com/trinodb)
+* Give us a star on the [Trino GitHub repository](https://github.com/trinodb/trino)
+* Follow us on the [Trino LinkedIn account](https://www.linkedin.com/company/trino-software-foundation)
+
+# Trino Summit 2022
+
+We hope you all join us in celebrating Trino’s birthday today. If you want to 
+learn even more, 
+[sign up for our hybrid event Trino Summit November 10th, 2022](https://www.starburst.io/info/trinosummit/).
```
Comment (mosabua) [inline _posts/2022-08-08-trino-tenth-birthday.md:91]:
```suggestion
[sign up for our hybrid event, Trino Summit, on the 10th of November 2022](https://www.starburst.io/info/trinosummit/).
```

**Review by mosabua — CHANGES_REQUESTED** [review]

**Pull request merged**